Kickstarter-iOS/Views/Controllers/ManageViewPledgeViewController.swift

```diff
@@ -221,6 +253,30 @@ final class ManageViewPledgeViewController: UIViewController {
   @objc private func closeButtonTapped() {
     self.dismiss(animated: true)
   }
+
+  // MARK: - Functions
+
+  private func goToRewards(_ project: Project) {
+    let rewardsVC = RewardsCollectionViewController.instantiate(with: project, refTag: nil)
+
+    self.navigationController?.pushViewController(rewardsVC, animated: true)
+  }
+
+  private func goToUpdatePledge(_: Project) {
```

Comment:
Are we omitting the variable name here because of SwiftFormat?

Reply:
Hmm good question I'm not sure actually. I'll add it back.

Reply:
Ok yeah I think because it's not being used SwiftFormat removes it. This'll be fixed once we consume the `project` argument.